mock/pubkeyConverterMock.go

```diff
@@ -0,0 +1,30 @@
+package mock
+
+import (
+	"encoding/hex"
+)
+
+// PubkeyConverterMock -
+type PubkeyConverterMock struct {
+	len int
```

Comment:
```suggestion
	Len int
```

So you can set it outisde this `mock` package